### PR TITLE
Update README.md with url

### PR DESCRIPTION
**Pull request by imagine-hussain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Circles
 
+You can find a live build of [Circles](https://circles.csesoc.app/degree-wizard) at [https://circles.csesoc.app/degree-wizard](https://circles.csesoc.app/degree-wizard).
```
Comment (gtangelo) [inline README.md]: Could we leave out the 'degree-wizard' route and just use the base url?

**Review by gtangelo — APPROVED** [review]
Just the change to the url. Otherwise lgtm!

File: README.md (hunk 1 above)
Comment (imagine-hussain) [reply README.md]: Sounds good.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Circles
 
+You can find a live build of [Circles](https://circles.csesoc.app/degree-wizard) at [https://circles.csesoc.app](https://circles.csesoc.app/).
```
Comment (Peedee2002) [inline README.md:3]: this is pedantic, but you probably dont want circles to be a hyperlink.

**Pull request merged**